rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -1348,24 +1344,6 @@ def wait(self, timeout=None):
         retcode = connector_binding.wait(self.native, timeout)
         _check_retcode(retcode)
 
-    @staticmethod
-    def set_max_objects_per_thread(value):
```

Comment:
We should deprecate the function and leave empty its implementation so we don't break existing applications.